src/dashboard/src/components/archival_storage/forms.py

```diff
@@ -17,15 +17,55 @@
 
 from django import forms
 
+from agentarchives.atom.client import CommunicationError
+from requests import Timeout, ConnectionError
+
+from components.archival_storage.atom import get_atom_client
+
 
 class CreateAICForm(forms.Form):
     results = forms.CharField(label=None, required=True, widget=forms.widgets.HiddenInput())
 
+
+class UploadMetadataOnlyAtomForm(forms.Form):
+    slug = forms.CharField(label='Insert slug', required=True, widget=forms.TextInput(attrs={'class': 'span8'}))
+
+    def clean_slug(self):
+        slug = self.cleaned_data['slug']
+        client = get_atom_client()
+        try:
+            client.find_parent_id_for_component(slug)
+        except (Timeout, ConnectionError) as e:
+            raise forms.ValidationError('Connection establishment failed: AtoM server cannot be reached.')
+        except CommunicationError as e:
+            if '404' in e.message:
+                raise forms.ValidationError('Description with slug %(slug)s not found!', code='notfound', params={'slug': slug})
+            raise forms.ValidationError('Unknown error: %(error)s', code='error', params={'error': e.message})
+        return slug
+
+
 class ReingestAIPForm(forms.Form):
     METADATA_ONLY = 'metadata'
     OBJECTS = 'objects'
     REINGEST_CHOICES = (
-        (METADATA_ONLY, 'Re-ingest metadata only'),
-        (OBJECTS, 'Re-ingest metadata and objects')
+        (METADATA_ONLY, 'Metadata only'),
+        (OBJECTS, 'Metadata and objects')
```

Comment:
I assume the full reingest option would come in with #384 / #391 with the rest of the feature.  If those get merged first, then this will have to be updated.